certbot/tests/util.py

```diff
@@ -276,6 +276,15 @@ def _write_msg(message, *unused_args, **unused_kwargs):
         """
         stdout.write(message)
 
+    def mock_method(*args, **kwargs):
+        """
+        Mock function for IDisplay methods.
+        """
+        message = args[0] if args else kwargs.get('message', None)
+        if message:
```

Comment:
If we want to avoid writing the message if it's empty, maybe we should do this in `_write_msg` itself?